src/commands/coverage/upload.ts

```diff
@@ -87,11 +111,17 @@ export class UploadCodeCoverageReportCommand extends Command {
   private measures = Option.Array('--measures')
   private tags = Option.Array('--tags')
   private format = Option.String('--format')
-  private skipGitMetadataUpload = Option.String('--skip-git-metadata-upload', 'false', {
+  private skipGitMetadataUpload = Option.String('--skip-git-metadata-upload', 'true', {
```

Comment:
this change is because git metadata is not used right now, right? If that's the case, I would simply remove it now and we can add it later. Other commands have this `skip` weirdness because of breaking changes, but this command is beta

Reply:
Makes sense, removed git upload altogether